src/components/Entry.vue

```diff
@@ -502,17 +536,18 @@ export default {
       );
       if (this.isExtraInputValidationRequired) {
         this.isCurrentUserValid = userValidationResponse;
+        this.validateCount = 2;
```

Comment:
This bypasses the second validation call. Will think of a more sophisticated method.

Reply:
Yes this is not good